Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -1736,6 +1740,14 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		56F327C70680D6E4A222CB8A /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				1A1225961174B8FB59FB139F /* MyPlayground.playground in Resources */,
```

Comment:
I don't think we should copy playgrounds into the target's bundle. Not sure if that should get fixed here, or if it should get fixed in a separate PR (first).

Reply:
It's a little unclear to me what's the desired behavior if someone decided to include the Swift playground on a target in `project.yml`... Let me check with daltonclaybrook and see what does he expect.

Reply:
@brentleyjones yeah so verified that this particular behavior was not a regression, see [here](https://github.com/yonaskolb/XcodeGen/issues/894#issuecomment-646376190). 

We should probably track that separately around improving resource copy control to avoid unexpected resources getting copied to the target?